filament/backend/include/backend/DriverEnums.h

```diff
@@ -130,24 +110,33 @@ enum class FenceStatus : int8_t {
 
 static constexpr uint64_t FENCE_WAIT_FOR_EVER = uint64_t(-1);
 
-static constexpr size_t SHADER_MODEL_COUNT = 3;
+/**
+ * Shader model
+ */
 enum class ShaderModel : uint8_t {
-    // For testing
+    //! For testing
     UNKNOWN    = 0,
-    // Mobile
+    //! Mobile
     GL_ES_30   = 1,
-    // Desktop
+    //! Desktop
```

Comment:
This might be a good place to note that we use these enums across all backends; they're not tied to GL specifically. Really they should be named more generically like `ShaderModel::MOBILE` and `ShaderModel::DESKTOP`.